addons/purchase/tests/test_purchase.py

```diff
@@ -946,3 +946,38 @@ def test_vendor_price_by_purchase_order_company(self):
         self.assertEqual(po.amount_untaxed, 15.0)
         po.company_id = company_a.id
         self.assertEqual(po.amount_untaxed, 10.0)
+
+    def test_bill_in_purchase_matching_individual(self):
+        """
+        Tests that if the vendor is an individual with a company, the bill will still appear when
+        using the purchase matching button on a vendor bill
+        """
+        company_partner = self.env['res.partner'].create({
+            'name': 'Small Company',
+            'company_type': 'company',
+        })
+        self.partner_a.parent_id = company_partner
+        purchase_order = self.env['purchase.order'].create({
+            'partner_id': self.partner_a.id,
+            'order_line': [(0, 0, {
```

Comment:
```suggestion
            'order_line': [Command.create({
```
For future PRs, [Commands](https://www.odoo.com/documentation/18.0/developer/reference/backend/orm.html#odoo.fields.Command) are easier to read :slightly_smiling_face: